DB/database.py

```diff
@@ -0,0 +1,43 @@
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy import Column, Integer, String, ForeignKey, DateTime
+from sqlalchemy.orm import relationship
+from sqlalchemy.ext.declarative import declarative_base
+import os
+from dotenv import load_dotenv
+load_dotenv()

```

Comment:
Expressions outside of a function are a bad habit. This code will be called every time the file is loaded and imported.   Read more here: https://www.geeksforgeeks.org/python-main-function/amp/